bot.js

```diff
@@ -764,76 +770,62 @@ bot.on('message', message => {
                 InvCommand(message, args);
-            }
-            else if(nameCmd === "regenland" || nameCmd === "rl"){
+            } else if (nameCmd === "regenland" || nameCmd === "rl") {
                 RegenLandCommand(message, args);
-            }
-            else if(nameCmd === "backup" && (message.author.id.toString() === '374929883698036736' || message.author.id.toString() === "542878436885266474")){
+            } else if (nameCmd === "backup" && (message.author.id.toString() === '374929883698036736' || message.author.id.toString() === "542878436885266474")) {
                 BackupCommand(message, args);
-            }
-            else if(nameCmd === "backup"){
+            } else if (nameCmd === "backup") {
                 message.reply("You have to be KeplerTeddy to do this command! Or someone who is verified that I choose");
                 //message.reply("This is better in DM.");
-            }
-            else if(nameCmd === "craft"){
+            } else if (nameCmd === "craft") {
                 CraftCommand(message, args);
-            }
-            else if(nameCmd === "multimine" || nameCmd === "mm"){
+            } else if (nameCmd === "multimine" || nameCmd === "mm") {
                 MultiMineCommand(message, args);
-            }
-            else if(nameCmd === "toplist" || nameCmd === "top"){
+            } else if (nameCmd === "toplist" || nameCmd === "top") {
                 TopListCommand(message, args);
-            }
-            else{
+            } else {
                 //message.reply("I have not heard of this command! Do kb!help to see the list of commands!");
             }
-        }
-        else if(maintenance === true){
+        } else if (maintenance === true) {
             //message.reply("The Kepler Bot is under maintenance! Try again later!");
         }
-    }
-    else{
+    } else {
         return;
     }
 });
 
 /** WHEN THE BOT IS READY DO THIS */
-bot.on('ready', function(){
-    if(maintenance){
-        bot.user.setActivity("Maintenance!!!", { type: 'LISTENING' })
-  .then(presence => console.log(`Activity set to ${presence.activity.name}`))
-  .catch(console.error);
-    }
-    else{
-        bot.user.setActivity("Kepler Bot | kb!help", { type: 'PLAYING' })
-  .then(presence => console.log(`Activity set to ${presence.activity.name}`))
-  .catch(console.error);
+bot.on('ready', () => {
+    if (maintenance) {
+        bot.user.setActivity("Maintenance!!!", {
+                type: 'LISTENING'
+            })
+            .then(presence => console.log(`Activity set to ${presence.activity.name}`))
+            .catch(console.error);
+    } else {
+        bot.user.setActivity("Kepler Bot | kb!help", {
+                type: 'PLAYING'
+            })
+            .then(presence => console.log(`Activity set to ${presence.activity.name}`))
+            .catch(console.error);
     }
     console.log("Bot is now on :P");
 });
 
-bot.login(process.env.TOKEN);
-
+bot.login(TOKEN);
```

Comment:
Added TOKEN variable since it was already an unused variable at the top on, the line `21`